.github/scripts/templates/author.md.jinja

```diff
@@ -1,67 +1,22 @@
----
-aliases:
-- {{author}}
-tags:
-- 
-publish: true
----
+{% include "authors/frontmatter.md.jinja" %}
 
-# {{author}}
 
-- GitHub: [{{user}}](https://github.com/{{user}}/) ^github
-<!-- - Discord: `@` ^discord-->
-{% if not website %}<!-- {% endif %}- Website: <{{ website }}> ^website{% if not website %}-->{% endif +%}
-<!-- - [[Publish sites|Publish site]]: ^publish-->
+{% include "authors/author_info.md.jinja" %}
 
-%% Feel free to add a bio below this comment %%
 
 
 ## Author of
 
-%% Begin Hub: Released contributions %%
-{% if plugins %}### Plugins
-{% for plugin in plugins %}
-- {{ plugin }}
-{% endfor %}
-{% endif %}
+{% include "authors/released_contributions.md.jinja" %}
 
```

Comment:
I'm not entirely sure whether this is one empty line more than before...

Reply:
Yes, the test does say the end result is identical. I'll reply in a comment in the PR, for visibility.